src/website/tracker/settings.py

```diff
@@ -336,6 +336,12 @@ def get_secret(name: str, encoding: str = "utf-8") -> str:
 
 GH_WEBHOOK_SECRET = get_secret("GH_WEBHOOK_SECRET")
 
+# When set to False, the application will escape package maintainers' name when
+# mentioning them in the issue body to avoid actually pinging them. This is used
+# as a safety measure during development. When the GitHub issue story is ironed
+# out, this should be set to True in production.
+GH_ISSUES_PING_MAINTAINERS = False
```

Comment:
Please add that to DjangoSettings (~l.46), otherwise it's really annoying to keep track of those options.

Reply:
Done in latest commit.